style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    gap: 1rem;
+    padding-top: 1rem;
+  }
+
+  .work-card .work-detials .tags .hash .HTML {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card .work-detials .tags .hash .CSS {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 45px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .work-card .work-detials .tags .hash .JavaScript {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 94px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .resume button {
+    box-sizing: border-box;
+    padding: 12px;
+    margin-top: 1.5rem;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 1.5rem;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    margin-right: 10rem;
+  }
+
+  .someter button {
+    width: 135px;
+    box-sizing: border-box;
+    height: 48px;
+    border: 5px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 1.5rem;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    margin: 1rem 1rem 10rem 10rem;
+    display: flex;
+    align-items: center;
+  }
+
+  .resume button:hover {
+    background-color: #396df2;
+    color: #fff;
+  }
+
+  .resume button:active {
+    background-color: #1b50d4;
+    color: #fff;
+  }
+
+  .resume button:disabled {
+    background-color: #fff;
+    color: grey;
+  }
+
+  #About .Aboutcontent .resume {
+    margin-top: 1rem;
+  }
+
+  .work-card .work-detials .submit button,
+  .work-card-3 .work-detials .submit button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    background-color: #fff;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
```

Comment:
- [ ] Kindly add a hover and active state to match the given template. USe the `:hover` and the `:active` pseudoclasses.
![image](https://user-images.githubusercontent.com/94551484/236449772-ba801235-e9b9-43ed-b1cc-d8d8c6f042ed.png)